smartcontract/service/native/ontid/utils.go

```diff
@@ -72,7 +72,7 @@ func decodeID(data []byte) ([]byte, error) {
 	return data[1:], nil
 }
 
-func verifyID(id []byte) bool {
+func VerifyID(id []byte) bool {
```

Comment:
remove this duplicated code and use the one in `account/identity.go`

Reply:
already done : )